### ISSUE TITLE
Config: "choose file" button unreachable by keyboard navigation

### ISSUE THREAD

**Issue by reynir** [robur-coop/dnsvizor#86]
> You can’t reach the “Choose File” button when using the keyboard. By using ‘tab’ on your keyboard, you should be able to cycle through all interactive elements on the page.

<img width="160" height="62" alt="Image" src="https://github.com/user-attachments/assets/ba654966-d249-418b-ac6f-86cdfe4fd5ab" />

[2.1.1 Keyboard](https://www.w3.org/WAI/WCAG22/Understanding/keyboard)
